BraintreeCore/src/androidTest/java/com/braintreepayments/api/BraintreeHTTPClientTest.java

```diff
@@ -24,18 +24,14 @@ public void setup() {
     @Test(timeout = 10000)
     public void getRequestSslCertificateSuccessfulInSandbox() throws InterruptedException, InvalidArgumentException {
         Authorization authorization = Authorization.fromString(Fixtures.TOKENIZATION_KEY);
-        BraintreeHttpClient braintreeHttpClient = new BraintreeHttpClient(authorization);
+        BraintreeHTTPClient braintreeHTTPClient = new BraintreeHTTPClient(authorization);
 
-        braintreeHttpClient.get("https://api.sandbox.braintreegateway.com/", null, new HttpResponseCallback() {
-            @Override
-            public void success(String responseBody) {
-                fail("Request should fail");
-            }
+        braintreeHTTPClient.get("https://api.sandbox.braintreegateway.com/", null, new HTTPResponseCallback() {
 
             @Override
-            public void failure(Exception exception) {
+            public void onResult(String responseBody, Exception httpError) {
```

Comment:
Should we assert that `responseBody` is null for these tests? Or an equivalent assertion to cover the previous fail("Request should fail") assertion on success?

Reply:
I think the assertions will fail properly. There were originally two code paths so the `fail()` would prevent a false positive where no assertions were invoked. Now that both result/error are in the same callback it should be cool.